synapse/lib/httpapi.py

```diff
@@ -418,7 +418,7 @@ async def getUseridenBody(self, validator=None):
 class WebSocket(HandlerBase, t_websocket.WebSocketHandler):
 
     async def xmit(self, name, **info):
-        await self.write_message(json.dumps({'type': name, 'data': info}))
+        await self.write_message(s_json.dumps({'type': name, 'data': info}).decode())
```

Comment:
Since `s_json.dumps()` returns a `bytes` object, decoding it to a string means that the tornado handlers have to re-encode that string as a `utf-8` bytestring before it is sent. Both websocket `write_message()` and handler `write()` methods are run through this function: https://github.com/tornadoweb/tornado/blob/v6.4.2/tornado/escape.py#L214-L224
```suggestion
        await self.write_message(s_json.dumps({'type': name, 'data': info}))
```